src/main.ts

```diff
@@ -325,8 +325,9 @@ export function generateDevelopmentCertificateFromCSR(
   // set certificate subject attrs from CSR
   certificate.setSubject(csr.subject.attributes);
 
-  // 30 day validity
+  // 30 day validity into the future, 5 days in the past just in case of clock skew at callsite
```

Comment:
I'd be more comfortable with a smaller value like 1 day instead of 5. Let's Encrypt uses 1 hour when calculating NotBeforeDate. Reading up on this, the purpose of NotBeforeDate is for when security vulnerabilities are found and people want to accept only certificates created after a fix is released (NotBeforeDate > FixDate).